crates/wash-lib/src/cli/registry.rs

```diff
@@ -99,4 +99,8 @@ pub struct RegistryPushCommand {
 
     #[clap(flatten)]
     pub opts: AuthOpts,
+
+    /// Push the artifact monolithically instead of chunked
+    #[clap(long = "monolithic", env = "WASH_PUSH_MONOLITHIC")]
```

Comment:
```suggestion
    #[clap(long = "monolithic-push", env = "WASH_MONOLITHIC_PUSH")]
```

I agree with the `WASH_SUBSECTION_*` pattern, but what made me feel this was weird was that the ENV var didn't exactly match the arg... IMO matching the arg is more important than subsection (mostly because we haven't been following that up until now)

Reply:
I was trying to avoid repetition in `wash push --monolithic-push` and `WASH_PUSH_MONOLITHIC` felt better than `WASH_MONOLITHIC_PUSH`, but I'm happy to go either way, appreciate the feedback!